src/charts/tooltip.js

```diff
@@ -401,15 +406,45 @@ define(function(require){
                 .attr('x2', tooltipWidth - 60);
         }
 
+        /**
+         * Calculates the number of lines the tooltip title will need and updates the
+         * initialTooltipBodyYPosition accordingly
+         */
+        function updateTooltipTitleYPosition() {
+            const approximateTitle = getTooltipTitle(Date.now());
+            const approximateNumberOfTitleLines = getApproximateNumberOfLines(
+                approximateTitle,
+                16,
+                tooltipMaxTitleLength
+            );
+
+            if(approximateNumberOfTitleLines > 1) {
+                additionalTooltipTitleHeight = 17 * (approximateNumberOfTitleLines -1)
+            }
+        }
+
         /**
          * Updates value of tooltipTitle with the data meaning and the date
          * @param  {Object} dataPoint Point of data to use as source
          * @return void
          * @private
          */
         function updateTitle(dataPoint) {
+            const tTitle = getTooltipTitle(dataPoint[dateLabel]);
```

Comment:
What does `tTitle` represent? I think a bit more descriptive name for this would be helpful for the reader. Is it `textTitle`?

Reply:
Tbh I don't know what it stands for, just sticked to the naming which was there before - so I did not introduce this variable. But it makes sense to leave it a little bit cleaner as it was before, changed it to `textTitle`, because most likely that's the meaning.